src/main/kotlin/no/nav/syfo/melding/kafka/legeerklaring/KafkaLegeerklaringDTO.kt

```diff
@@ -147,3 +151,32 @@ data class Signatur(
     val signatur: String?,
     val tlfNummer: String?,
 )
+
+fun KafkaLegeerklaringDTO.toMeldingFraBehandler() =
+    MeldingFraBehandler(
+        uuid = UUID.randomUUID(),
+        createdAt = OffsetDateTime.now(),
+        type = MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING,
```

Comment:
Litt usikker, men virker riktigst å bruke samme type som for den utgående meldingen. Den vil jo ha innkommende=true, så det burde være greit å skille den fra den utgående meldingen.

Reply:
Tror det blir riktig ja 👍